prisma/migrations/20241206160747_update_build_prices_names/migration.sql

```diff
@@ -0,0 +1,13 @@
+/*
+  Warnings:
+
+  - You are about to drop the `buildprices` table. If the table is not empty, all the data it contains will be lost.
```

Comment:
Should warnings be removed after the query is updated to use `RENAME` instead of `DROP`?

Reply:
Good spot thanks - yes! We should keep relevant warnings (very good for PRs) but remove redundant/incorrect ones.

Reply:
Please see ced8b008d76d8de52e3701e8633de730d6e7b107